week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -13,7 +13,12 @@
   
 */
 
-function colonisers() {}
+function colonisers(arr) {
+  const newArray = arr.filter(
+    (word) => word.startsWith("A") && word.includes("family")
+  );
+  return newArray;
+}
```

Comment:
This is correct, but a slightly shorter way of writing this is:
function colonisers(arr) {
  return  arr.filter(
    (word) => word.startsWith("A") && word.includes("family")
  );
}